### ISSUE TITLE
Agreeing list of 'features'

### ISSUE THREAD

**Issue by aidanbudd** [aidanbudd/bosc2015#10]
OK now with the list of features?

It might be too long, but I struggle to shorten it by merging/removing redundancy - some features are similar/related, but still I find valuably distinct, and don't want to loose that.

I like that several features are software-specific, i.e. it's not just a list of features of great communities in general.

Only a few of us contributed so far - perhaps if we move now to start collecting specific examples/anecdotes/evidence from specific projects, to support these features as important, more people can more easily contribute?

Let me know which of these you think's the best way to move forward:
1. discuss (by mail? here in github's issues?) and agree together changes to current set of features
2. move on now to collecting examples from specific OSBC's we've been involved in, and reaching out/inviting others to contribute

**Comment by bgruening** [comment]
I'm voting for both and here in public :)

**Comment by malvikasharan** [comment]
I agree with Björn. Both the options should be open for some people might find it more comfortable writing to you than directly here.

**Comment by ghost** [comment]
I agree, we need more people to contribute. The good idea would be to post about this on Biostars, & repost it on Twitter. In terms of discussion, mail and github sound absolutely fine. Another option is Google docs but I understand that some people may not prefer it.

**Comment by aidanbudd** [comment]
Alrighty (in the words of Ace Ventura) - thanks for your comments. In that case, let's:
- post this on biostars and re-tweet it - given you suggested it, Fatima, I encourage you to do that
- if you've specific examples to add to illustrate the points already listed, go to it and add them
- if there are some of the points you think it'd be good to merge/adapt/etc. , let us know here and/or pull request away
- if there's anyone you think would like to get involved, mail 'em to encourage them to do so

**Comment by aidanbudd** [comment]
Have tried clustering features into broad categories. Inevitably, some (most?) features could be placed in multiple categories. Please go ahead and try out your own clustering if you can see ways to improve it!

**Comment by aidanbudd** [comment]
Just posted to the BioStar forum section encouraging people to send us their contributions to this https://www.biostars.org/p/153770/

**Comment by mr-c** [comment]
One could easily cross reference the features and sub features with the Software Sustainability Institute's criteria based software evaluation guide http://www.software.ac.uk/software-evaluation-guide

**Comment by aidanbudd** [comment]
Thanks  a lot for that, Michael. Lots of overlap with the points in that document; when I've a 'mo, I'll have a go at integrating (and referencing) these points into the list here. And/or, of course, if someone else wants to do that (Michael...? :) ) more than happy for you to edit listOfFeaturesOfGreatOSBCs.md and send a pull request!

**Comment by aidanbudd** [comment]
A Doodle poll to find time to meet online for several hours to try and agree together a potentially-finalish conciser-than-currently list of features http://doodle.com/h78f4adxbgpt2csm

**Comment by aidanbudd** [comment]
We agreed to meet 14.00 on Thursday 10th September 2015, German time (CEST http://www.timeanddate.com/time/zones/cest), which is 13.00 UK time (BST http://www.timeanddate.com/time/zones/bst)/. This time works for everyone who's filled in the poll http://doodle.com/h78f4adxbgpt2csm

**Comment by aidanbudd** [comment]
Sorry for the delay in getting back to this - getting my contract renewed kept me kinda distracted, off sick, the normal RL stuff :)

Just now checked out the googledoc that was the result of the previous GoogleHangout Sep 10th 2015, thinking about what to do next.

To move on from here I think we need to do the following things next:

(A) and (B) can be done in parallel.
- (A) move the text from the googledoc into markdown (putting the comments in using HTML comment tags) to replace the current listOfFeaturesOfGreatOSBCs.md
  - (A.1) once it's in Markdown, divide up the task of summarising and standardising the text associated with each of the different features
- (B) devising a list of three questions to contact members of communities we're interested in including in the manuscript with, to get their anecdotes on how they've dealt with building their OS scientific communities (B.1) and (B.2) can be done in parallel
  - (B.1) testing these questions ourselves by answering them for communities we're involved in
  - (B.2) choosing a list of priority communities for us to contact for feedback 

Once we've got that done, we can reach out to these communities and work out how to build their content/answers to these questions into the manuscript.

Would any of you like to start going with any of that? I'll start on it myself, but would be quicker, more in the spirit of what we're trying to do, and I expect much better, if some of you were up for doing some of these things.

I'll mail all the contributors so far now, to let you know about this post, so hopefully we can start moving with this again.

**Comment by IsmailM** [comment]
For A1 - I’ll get that done in a moment… 

I’ve used this [Google script](https://github.com/mangini/gdocs2md) before for converting google docs into markdown, and it worked quite well at the time.

I’ll try it out and submit a PR in a moment - hopefully it should be good enough for this…

**Comment by aidanbudd** [comment]
Thanks for that @IsmailM - this gave me a chance to train my merge-conflict resolution skills, as I did something similar ('manually' rather than using a script) while in the air. Happily resolved now :)
